src/b_setup.hpp

```diff
@@ -421,7 +423,8 @@ void setup()
     mount.configureFocusStepper(FOCUSmotorPin1, FOCUSmotorPin2, FOCUS_STEPPER_SPEED, FOCUS_STEPPER_ACCELERATION);
     #endif
     #if FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
-    LOGV1(DEBUG_ANY, F("Configure Focus driver..."));
+    LOGV1(DEBUG_ANY, F("setup(): Configure Focus driver..."));
```

Comment:
Should this still be `DEBUG_ANY`?

Reply:
Yes, all the configuration logs for this line are DEBUG_ANY